src/mapToMetadataTree.js

```diff
@@ -18,20 +18,25 @@ require('./typedef');
  * HTML of the section
  */
 function parseSectionTokens(links, tokens) {
+  if (!tokens || tokens.length == 0) return [{}, ''];
+
+  const listIdx = tokens.findIndex(token => token.type == 'list');
+  if (listIdx < 0) return [{}, ''];
+
+  const list = tokens[listIdx];
+  if (!list || !list.items) return [{}, ''];
```

Comment:
Lines 24 and 27 are not covered by the existing tests. Please could you add a test case?